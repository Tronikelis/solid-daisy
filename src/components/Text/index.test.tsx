import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Text } from ".";

describe("<Text />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Text>foo</Text>);
        expect(container).toMatchSnapshot();
    });
});
