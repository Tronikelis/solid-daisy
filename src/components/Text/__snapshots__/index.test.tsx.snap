// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Text /> > renders correctly 1`] = `
<div>
  <p
    class="text-base"
  >
    foo
  </p>
</div>
`;
